zuul.d/job.yaml

```diff
@@ -1,11 +1,8 @@
 ---
 - job:
-    name: tcib-build-containers
-    parent: cifmw-tcib-base
-    nodeset: centos-stream-9-vexxhost
-    vars:
-      cifmw_build_containers_registry_namespace: podified-antelope-centos9
-      cifmw_build_containers_image_tag: "{{ cifmw_repo_setup_full_hash }}"
+    name: tcib-podified-multinode-edpm-deployment-crc
+    parent: podified-multinode-edpm-deployment-crc
+    dependencies: ["openstack-meta-content-provider"]
```

Comment:
maybe better to use the explicit list syntax here for readability?
dependencies:
    - openstack-meta-content-provider

Reply:
Done